parsers/nmparser/parse_lst_file.go

```diff
@@ -107,11 +291,31 @@ func ParseLstEstimationFile(lines []string) ModelOutput {
 			shrinkageDetails = parseShrinkage(line, shrinkageDetails)
 		case strings.Contains(line, "EPSSHRINK"):
 			shrinkageDetails = parseShrinkage(line, shrinkageDetails)
+		case strings.Contains(line, "0MINIMIZATION SUCCESSFUL"):
+			runHeuristics.MinimizationSuccessful = HeuristicTrue
+		case strings.Contains(line, "GRADIENT:"):
+			gradientLines = append(gradientLines, line)
+		case strings.Contains(line, "RESET HESSIAN"):
+			runHeuristics.HessianReset = HeuristicTrue
+		case strings.Contains(line, "PARAMETER ESTIMATE IS NEAR ITS BOUNDARY"):
+			runHeuristics.ParameterNearBoundary = HeuristicTrue
+		case strings.Contains(line, "COVARIANCE STEP OMITTED: NO"):
+			runHeuristics.CovarianceStepOmitted = HeuristicTrue
+		case strings.Contains(line, "EIGENVALUES OF COR MATRIX OF ESTIMATE"):
+			// TODO: get largeNumberLimit from config
+			// or derive. something like (number of parameters) * 10
+			runHeuristics.LargeConditionNumber = getLargeConditionNumberStatus(lines, i, 1000.0)
+		case strings.Contains(line, "CORRELATION MATRIX OF ESTIMATE"):
+			// TODO: get correlationLimit from config
+			runHeuristics.CorrelationsOk = getCorrelationStatus(lines, i, 0.95)
```

Comment:
how did you choose a value of 0.95?

Reply:
PsN default it 0.9, so I'll change to that.

https://github.com/UUPharmacometrics/PsN/blob/88320646310b63f0c88075a6a71f4d5a87be4528/bin/sumo#L59

Also, from sumo doc:
-correlation_limit = N
Default is 0.9. The cut-off for issuing a warning for high correlations between the parameter estimates. This test will only
be made if the $COV step has been requested in the NMtran
control stream.
The default is set in psn.conf.